src/System.Private.Uri/tests/FunctionalTests/UriTests.cs

```diff
@@ -731,13 +734,16 @@ public static void TestIsWellFormedOriginalString()
             b = uri.IsWellFormedOriginalString();
             Assert.False(b);
 
-            uri = new Uri(@"c:\\directory\filename");
-            b = uri.IsWellFormedOriginalString();
-            Assert.False(b);
+            if (s_IsWindowsSystem)
+            {
+                uri = new Uri(@"c:\\directory\filename");
+                b = uri.IsWellFormedOriginalString();
+                Assert.False(b);
 
-            uri = new Uri(@"file://c:/directory/filename");
-            b = uri.IsWellFormedOriginalString();
-            Assert.False(b);
+                uri = new Uri(@"file://c:/directory/filename");
+                b = uri.IsWellFormedOriginalString();
+                Assert.False(b);
+            }
```

Comment:
> Technically it is possible. I doubt whether it will be used.

It may not be, or at least not on purpose.  But since such code would have worked on desktop and on .NET Core 1.x on Unix, if we can support it, it could be good, too.  What does Mono do with such paths on Unix?